tests/Sinch.Tests/e2e/Verification/VerificationStartTests.cs

```diff
@@ -108,27 +108,26 @@ public async Task StartFlashCallVerification()
         }
 
         [Fact]
-        public async Task StartPhoneCallVerification()
+        public async Task StartCalloutVerification()
         {
             var startVerificationRequest = new StartVerificationRequest
             {
                 Identity = _identity,
                 Method = VerificationMethodEx.Callout
             };
-            var startPhoneCallVerificationResponse = new StartCalloutVerificationResponse
-            {
-                Id = _id,
-                Method = VerificationMethodEx.Callout,
-                Links = _links
-            };
 
             var response = await VerificationClient.Verification.StartCallout(new StartCalloutVerificationRequest
             {
                 Identity = startVerificationRequest.Identity,
                 Reference = startVerificationRequest.Reference,
                 Custom = startVerificationRequest.Custom
             });
-            response.Should().BeEquivalentTo(startPhoneCallVerificationResponse);
+            response.Should().BeEquivalentTo(new StartCalloutVerificationResponse
```

Comment:
Nit: you may recreate a variable `startCalloutVerificationResponse ` as it used to be and as it is in the other tests for consistent reading